sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -69,6 +69,7 @@ object Klaviyo {
         Registry.get<ApiClient>().startService()
 
         Registry.register<State>(KlaviyoState())
+        Registry.getOrNull<StateSideEffects>()?.detach()
```

Comment:
Because we didn't detach previous listeners, we risked accumulating multiple identical side effect listeners 😬